charts/internal/machine-controller-manager/seed/templates/vpa.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.vpa.enabled }}
-apiVersion: autoscaling.k8s.io/v1beta2
```

Comment:
while we're here: we should probably also adapt the `apiVersion` for this one https://github.com/gardener/gardener-extension-provider-equinix-metal/blob/master/charts/internal/seed-controlplane/charts/cloud-provider-equinix-metal/templates/vpa.yaml#L1

Not necessarily in this PR, I'm fine with doing this in a separate one, if you want.

Reply:
Done